src/swagger-specs/segmentation.yaml

```diff
@@ -924,8 +924,7 @@ paths:
         - $ref: '#/components/parameters/x-sandbox-name'
         - name: x-request-id
           in: header
-          description: There needs to be one unique ID per request.
-          required: true
+          description: You can use a unique ID to track the request.
```

Comment:
Should this be 'this' instead of 'a'? As in:
"You can use this unique ID to track the request."